packages/utils/interactive-hooks/src/useKeyProps.ts

```diff
@@ -23,38 +29,106 @@ const noOp = () => {};
  * @returns onKeyEvent() - Callback to determine if key was pressed, if so, call userCallback
  * @deprecated use the hook `useKeyProps` instead
  */
-export function useKeyCallback(_userCallback?: KeyCallback, ..._keys: string[]) {
-  return noOp;
+export function useKeyCallback(userCallback?: KeyCallback, ...keys: string[]) {
+  const onKeyEvent = React.useCallback(
+    (e: KeyPressEvent) => {
+      if (userCallback !== undefined && (keys === undefined || keys.includes(e.nativeEvent.key))) {
+        userCallback(e);
+        e.stopPropagation();
+      }
+    },
+    [keys, userCallback],
```

Comment:
Worth noting this used to be `...keys` and eslint complained, so I updated. That is _probably_ a functionality change.

Reply:
What exactly was eslint complaining about?

Reply:
It can't statically check a spread operation, so it asked me to use "keys" instead of "...keys"